Sources/JExtractSwiftLib/Swift2Java.swift

```diff
@@ -26,12 +26,12 @@ public struct SwiftToJava {
   }
 
   public func run() throws {
-    guard let swiftModule = config.swiftModule else {
+    guard let moduleName = config.swiftModule else {
```

Comment:
You're changing "module name" to "swift module", but this is opposite? 🤔

Reply:
Good catch, I probably resolved a conflict the wrong way